README.md

```diff
@@ -1,19 +1,26 @@
+[integration-tests-badge]: https://github.com/conda-incubator/conda-recipe-manager/actions/workflows/integration_tests.yaml/badge.svg?branch=main
+[release-badge]: https://img.shields.io/github/v/release/conda-incubator/conda-recipe-manager?logo=github
+
+
 # conda-recipe-manager
 
+[![Integration Tests][integration-tests-badge]](https://github.com/conda-incubator/conda-recipe-manager/actions/workflows/integration_tests.yaml)
```

Comment:
Should we add a marker for the unit tests/commit checks? I guess that would basically be the code coverage badge you mentioned.

Reply:
Yep, we can do that!